sys/net/application_layer/nanocoap/Makefile

```diff
@@ -0,0 +1 @@
+include $(RIOTBASE)/Makefile.base
```

Comment:
Right now `nanocoap_sock.c` is always build, but not always needed (e.g. for `gcoap`), right? I think it would make much sense to make `nanocoap_sock` a sub-module of `nanocoap`, so that I must be enabled explicitly by using `USEMODULE += nanocoap_sock` e.g. in the example application above. Makes sense?

Reply:
ok